packages/core/src/domain/session/sessionStore.ts

```diff
@@ -153,19 +182,10 @@ export function startSessionStore<TrackingType extends string>(
 
   function retrieveActiveSession(): SessionState {
     const session = retrieveSession()
-    if (isActiveSession(session)) {
+    if (!isSessionInExpiredState(session)) {
       return session
     }
-    return {}
-  }
-
-  function isActiveSession(sessionState: SessionState) {
-    // created and expire can be undefined for versions which was not storing them
-    // these checks could be removed when older versions will not be available/live anymore
-    return (
-      (sessionState.created === undefined || dateNow() - Number(sessionState.created) < SESSION_TIME_OUT_DELAY) &&
-      (sessionState.expire === undefined || dateNow() < Number(sessionState.expire))
-    )
+    return getInitialSessionState()
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏`retrieveActiveSession` seems a bit odd now, could `retrieveStartedSession` be more accurate?